packages/ogmios/test/CardanoNode/mappers.test.ts

```diff
@@ -1,26 +0,0 @@
-import * as mappers from '../../src/CardanoNode/mappers';
-import { EraSummary } from '@cardano-sdk/core';
-import { Schema } from '@cardano-ogmios/client';
-
-describe('cardano node mappers', () => {
-  describe('mapEraSummary', () => {
```

Comment:
This test is still useful as it transforms `time` into `Date`

Reply:
Brought it back in 036ff826f617b8d0d6f937ef2e729028061dd081

Reply:
Good catch. I misinterpreted the test intent. Will suggest a change to fix the name